stubs/Flask-Cors/flask_cors/core.pyi

```diff
@@ -0,0 +1,41 @@
+from logging import Logger
+from typing import Any, Iterable, Pattern, TypeVar
+
+_T = TypeVar("_T")
+_App = Any  # flask is not part of typeshed
+_Response = Any  # flask is not part of typeshed
+_Options = dict[str, Any]
```

Comment:
This could probably be a TypedDict.

Reply:
So the options are the same as e.g. for CORS.__init__. Is there a way to avoid duplicate definitions?

Reply:
Unfortunately not.